std_msgs/Header header
uint32 RequestId
bool mouseClicksAreAccepted